tests/services/IApiService.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiContract } from "../../src/contracts/api";
+export interface IApiService {
```

Comment:
May be better to named ITestApiService to avoid mixing with not testing services?

Reply:
thank you for review